common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -424,6 +424,16 @@ def editor_saved(self, user, old_metadata, old_content):
         This should be fixed too.
         """
         metadata_was_changed_by_user = old_metadata != own_metadata(self)
+        # If metadata is not changed by user and html5_sub does not exist.
+        if self.sub and old_metadata.get('html5_sources', []) and not metadata_was_changed_by_user:
```

Comment:
why `old_metadata.get('html5_sources', [])`? Why not `old_metadata.get('html5_sources', False)` or just `old_metadata.get('html5_sources')`